plugins/osdn/ovs/plugin.go

```diff
@@ -54,8 +55,13 @@ func (plugin *ovsPlugin) PluginStartMaster(clusterNetwork *net.IPNet, hostSubnet
 		if err := plugin.VnidStartMaster(); err != nil {
 			return err
 		}
+	} else {
+		// Network plugin may be switched from multi-tenant to single-tenant
+		// Delete VNID annotations on the namespaces if needed
+		if err := plugin.VnidCleanup(); err != nil {
```

Comment:
An admin who wants to fix this up can run `oc annotate VNIDANNOTATION- namespaces --all` if they really want